### DISCUSSION TITLE
Account Blocked

### DISCUSSION THREAD

**Discussion by multilinks50** [vercel/community#4218] [Account Issue] [answered]
### Question

I want to deploy my webiste but vercel blocked the github account.

**Comment by swarnava** [comment, marked as answer]
Hi,

Please email us at registration@vercel.com

**Reply by multilinks50** [reply]
ok i have send mail on this ..kindly check

**Reply by multilinks50** [reply]
i have already send 2 mails in this address but got no response

**Reply by swarnava** [reply]
Hi,

Our team will get back to you! :)

**Reply by hungchihsueh** [reply]
@swarnava  hello, I sent the email to regisration@vercel.com couple week ago and still get no response, is there anything i can do?
I have 2 domain on my personal account I really need help to get it back

**Comment by multilinks50** [comment]
ok thank you! for looking into it :)
